app/DynamicsAdapter/Fams3Adapter.Dynamics/Identifier/InformationSourceType.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Fams3Adapter.Dynamics.OptionSets.Models;
+
+namespace Fams3Adapter.Dynamics.Identifier
+{
+    public class InformationSourceType : Enumeration
+    {
+
+        public static InformationSourceType DriverLicense = new InformationSourceType(867670000, "Request");
```

Comment:
why DriverLicence = Request ?

Reply:
good catch. copy paste, then forget to change it. :(